src/models/db/transfer.js

```diff
@@ -72,11 +69,44 @@ class Transfer extends Model {
   static convertFromPersistent (data) {
     delete data.created_at
     delete data.updated_at
+    if (typeof data.credits === 'string') {
+      data.credits = JSON.parse(data.credits)
+    }
+    if (typeof data.debits === 'string') {
+      data.debits = JSON.parse(data.debits)
+    }
+    if (typeof data.execution_condition === 'string') {
+      data.execution_condition = JSON.parse(data.execution_condition)
+    }
+    if (typeof data.cancellation_condition === 'string') {
+      data.cancellation_condition = JSON.parse(data.cancellation_condition)
+    }
+    if (typeof data.additional_info === 'string') {
+      data.additional_info = JSON.parse(data.additional_info)
+    }
+    if (data.expires_at) {
+      data.expires_at = new Date(data.expires_at)
```

Comment:
I think because the standard format is a date object. So when you get it from the database you have to parse to a date, but when putting it into the database it automatically converts the date to the proper format.